server/src/main/java/keywhiz/service/daos/SecretSeriesDAO.java

```diff
@@ -185,6 +185,10 @@ public int setCurrentVersion(long secretId, long secretContentId, String updater
     return Optional.ofNullable(r).map(secretSeriesMapper::map);
   }
 
+  public List<SecretSeries> getSecretsSeriesByName(List<String> names) {
```

Comment:
nit: maybe name this `getMultipleSecretSeriesByName` to make the API difference clearer--I think it'll be easy to miss the extra `s` in `SecretsSeries` versus `SecretSeries`

Reply:
Good call!